pysat/tests/test_utils.py

```diff
@@ -260,3 +427,50 @@ def test_write_and_read_netcdf4_default_format_higher_order_w_zlib(self):
 
         assert (np.all((test_inst.data == loaded_inst).all()))
         assert np.all(test_list)
+
+    def test_netcdf_attribute_override(self):
+        """Test that attributes in netcdf file may be overridden"""
+        self.testInst.load(2009, 1)
+
+        try:
+            assert self.testInst.bespoke # should raise
+        except AttributeError:
+            pass
```

Comment:
What is this testing?  What should it be raising?  Can this be tested using `raises`?

Reply:
The specific lines are a pre-test to ensure a proper setup, that the attribute testInst.bespoke does not exist. The next lines set testInst.meta.bespoke and it is transferred to inst using a method. Then we test that testInst.bespoke is True. If so, we know it came from meta.